docs/content/installation/installing-nic/installation-with-manifests.md

```diff
@@ -58,40 +58,67 @@ This guide assumes you are using the latest release.
 
 To make sure your NGINX Ingress Controller pods reach the `Ready` state, you'll need to create custom resource definitions (CRDs) for various components. Alternatively, you can disable this requirement by setting the `-enable-custom-resources` command-line argument to `false`.
 
+{{<tabs name="crd-install">}}
+
+{{%tab name="Install CRDs from single YAML"%}}
+
 ### Core custom resource definitions
 
-1. Create CRDs for [VirtualServer and VirtualServerRoute]({{< relref "configuration/virtualserver-and-virtualserverroute-resources.md" >}}), [TransportServer]({{< relref "configuration/transportserver-resource.md" >}}), and [Policy]({{< relref "configuration/policy-resource.md" >}}):
+1. Create CRDs for [VirtualServer and VirtualServerRoute]({{< relref "configuration/virtualserver-and-virtualserverroute-resources.md" >}}), [TransportServer]({{< relref "configuration/transportserver-resource.md" >}}), [Policy]({{< relref "configuration/policy-resource.md" >}}) and [GlobalConfiguration]({{< relref "configuration/global-configuration/globalconfiguration-resource.md" >}}):
 
     ```shell
-    kubectl apply -f common/crds/k8s.nginx.org_virtualservers.yaml
-    kubectl apply -f common/crds/k8s.nginx.org_virtualserverroutes.yaml
-    kubectl apply -f common/crds/k8s.nginx.org_transportservers.yaml
-    kubectl apply -f common/crds/k8s.nginx.org_policies.yaml
+    kubectl apply -f https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v3.3.2/deploy/crds.yaml
     ```
 
 ### Optional custom resource definitions
 
-1. (Optional) For TCP and UDP load balancing, create a cCRD for [GlobalConfiguration]({{< relref "configuration/global-configuration/globalconfiguration-resource.md" >}}):
+1. For the NGINX App Protect WAF module, create CRDs for `APPolicy`, `APLogConf` and `APUserSig`:
+
+    ```shell
+    kubectl apply -f https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v3.3.2/deploy/crds-nap-waf.yaml
+    ```
+
+2. For the NGINX App Protect DoS module, create CRDs for `APDosPolicy`, `APDosLogConf` and `DosProtectedResource`:
+
+    ```shell
+    kubectl apply -f https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v3.3.2/deploy/crds-nap-dos.yaml
+    ```
+
+{{%/tab%}}
+
+{{%tab name="Install CRDs from cloning the repo"%}}
```

Comment:
If this path requires a user to clone the repo first, we should add that as a step here and include recommendations as to the path where the repo should be cloned if applicable.